docs/sphinxext/mantiddoc/directives/categories.py

```diff
@@ -348,14 +348,20 @@ def create_category_pages(app):
         # Now any additional index pages if required
         if category.name in INDEX_CATEGORIES:
             # index in categories directory
-            category_html_dir = os.path.dirname(category.html_path)
-            category_html_path_noext = category_html_dir + "/index"
+            if category.name == 'Concepts':
```

Comment:
I think this can be more general. If a page in API would point to category of fit functions, that would be shown in API